torch/backends/cudnn/__init__.py

```diff
@@ -194,6 +211,11 @@ def __init__(self, m, name):
     allow_tf32 = ContextProp(
         torch._C._get_cudnn_allow_tf32, torch._C._set_cudnn_allow_tf32
     )
+    conv = FP32Precision("cuda", "conv")
```

Comment:
No "matmul" flag here?

Reply:
matmul is under `cuda`, not `cudnn`. We do not wish to change the original structures.